### PR TITLE
Additional task field: is_private

### PR DESCRIPTION
**Pull request by elioschmutz**
Dieser PR implementiert das neue Feld für Aufgaben: `is_private`

bildschirmfoto 2018-10-10 um 13 54 12

Issuer: #4763

### CODE REVIEW

**Comment by elioschmutz** [comment]
Something is weird with this PR. The commits are not correctly displayed. I can't fix it with a forcepush.

I opened a new PR with a new Branch: #4957

File: opengever/task/localroles.py [hunk 1]
```diff
@@ -80,6 +80,12 @@ def _add_local_roles(self, context, principal, roles, is_agency=False):
 
         RoleAssignmentManager(context).add_or_update_assignment(assignment)
 
+    def _should_add_agency_localroles(self):
```
[the remark this answers is not in the document]
Comment (elioschmutz) [reply opengever/task/localroles.py]: What do you mean exactly? This function only extends the current implementation with the `is_private` property and handles only adding localroles for tasks.

I don't see a need for handling revoking roles... or do I misunderstand something?

File: opengever/task/task.py [hunk 2]
```diff
@@ -158,6 +160,15 @@ class ITask(model.Schema):
 
     form.widget(deadline=DatePickerFieldWidget)
 
+    is_private = schema.Bool(
```
[the remark this answers is not in the document]
Comment (elioschmutz) [reply opengever/task/task.py]: 👍

**Comment by Rotonen** [comment]
@elioschmutz commit timestamps vs. authoring timestamps - that's normal, expected and not that critical.

**Pull request closed without merging**